store/store.go

```diff
@@ -183,6 +180,31 @@ func (s *Store) SyncDiskKeyStoreToDb() error {
 	return merr
 }
 
+// ImportDatabaseKeys writes all keys stored in the orm to the keystore
+// keys directory.
+func (s *Store) ImportDatabaseKeys(pwd string) error {
+	keysDir := s.Config.KeysDir()
+	if err := os.RemoveAll(keysDir); err != nil {
+		return err
+	}
+
+	if err := os.MkdirAll(keysDir, 0700); err != nil {
+		return err
+	}
+
+	keys, err := s.Keys()
+	if err != nil {
+		return err
+	}
+
+	var merr error
+	for _, k := range keys {
+		_, err := s.KeyStore.Import([]byte(k.JSON.String()), pwd, pwd)
```

Comment:
I thought `KeyStore.Import` was the better way to do it, since it works with go ethereum's internal caching mechanism, the Import recreates the cipher, salt, etc resulting in a different file every boot. Is that ok?

```
--- FAIL: TestStore_ImportDatabaseKeys (1.93s)
    store_test.go:186:
        	Error Trace:	store_test.go:186
        	Error:      	Not equal:
        	            	expected: map[string]interface {}{"address":"3cb8e3fd9d27e39a5e9e6852b0e96160061fd4ea", "crypto":map[string]interface {}{"cipher":"aes-128-ctr", "ciphertext":"7515678239ccbeeaaaf0b103f0fba46a979bf6b2a52260015f35b9eb5fed5c17", "cipherparams":map[string]interface {}{"iv":"87e5a5db334305e1e4fb8b3538ceea12"}, "kdf":"scrypt", "kdfparams":map[string]interface {}{"n":262144, "p":1, "r":8, "salt":"d89ac837b5dcdce5690af764762fe349d8162bb0086cea2bc3a4289c47853f96", "dklen":32}, "mac":"57a7f4ada10d3d89644f541c91f89b5bde73e15e827ee40565e2d1f88bb0ac96"}, "id":"c8cb9bc7-0a51-43bd-8348-8a67fd1ec52c", "version":3}
        	            	actual  : map[string]interface {}{"version":3, "address":"3cb8e3fd9d27e39a5e9e6852b0e96160061fd4ea", "crypto":map[string]interface {}{"cipher":"aes-128-ctr", "ciphertext":"bf4537b36cd4078f9c315b0d4e0a76ce721c12a0e99c8b002692fd6e02e01ba7", "cipherparams":map[string]interface {}{"iv":"ca606b3d37f78f2bd6734b1dfc26bd42"}, "kdf":"scrypt", "kdfparams":map[string]interface {}{"dklen":32, "n":262144, "p":1, "r":8, "salt":"eb7e8877402ef4049c702c2aa3b1c6416b864d843a6591d2ce1daaabfa9e840d"}, "mac":"723009a41d0157ffb806ff94311579c016f743775a82239f32f1911c745921c2"}, "id":"c8cb9bc7-0a51-43bd-8348-8a67fd1ec52c"}

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -5,5 +5,5 @@
        	            	   (string) (len=12) "cipherparams": (map[string]interface {}) (len=1) {
        	            	-   (string) (len=2) "iv": (string) (len=32) "87e5a5db334305e1e4fb8b3538ceea12"
        	            	+   (string) (len=2) "iv": (string) (len=32) "ca606b3d37f78f2bd6734b1dfc26bd42"
        	            	   },
        	            	-  (string) (len=10) "ciphertext": (string) (len=64) "7515678239ccbeeaaaf0b103f0fba46a979bf6b2a52260015f35b9eb5fed5c17",
        	            	+  (string) (len=10) "ciphertext": (string) (len=64) "bf4537b36cd4078f9c315b0d4e0a76ce721c12a0e99c8b002692fd6e02e01ba7",
        	            	   (string) (len=3) "kdf": (string) (len=6) "scrypt",
        	            	@@ -14,5 +14,5 @@
        	            	    (string) (len=1) "r": (float64) 8,
        	            	-   (string) (len=4) "salt": (string) (len=64) "d89ac837b5dcdce5690af764762fe349d8162bb0086cea2bc3a4289c47853f96"
        	            	+   (string) (len=4) "salt": (string) (len=64) "eb7e8877402ef4049c702c2aa3b1c6416b864d843a6591d2ce1daaabfa9e840d"
        	            	   },
        	            	-  (string) (len=3) "mac": (string) (len=64) "57a7f4ada10d3d89644f541c91f89b5bde73e15e827ee40565e2d1f88bb0ac96"
        	            	+  (string) (len=3) "mac": (string) (len=64) "723009a41d0157ffb806ff94311579c016f743775a82239f32f1911c745921c2"
        	            	  },
        	Test:       	TestStore_ImportDatabaseKeys
FAIL
FAIL	github.com/smartcontractkit/chainlink/store	2.217s
```

Reply:
Reverted back to the simpler disk copy over `KeyStore.Import` because the reimporting mutates the cipher, salt, etc.